### PR TITLE
[PM-7057] Add dev tool for toggling width in web vault

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add a dev tool for toggling the web vault width. This allows developers and designers to experiment with how the vault currently behaves with responsiveness and ensure new functionality looks good.

## Screenshots

https://github.com/bitwarden/clients/assets/137855/dedabf12-97dc-4be6-8c0f-b625b222d124

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/app/layouts/toggle-width.component.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ></bit-nav-item>`,
+  standalone: true,
+  imports: [CommonModule, NavigationModule],
+})
+export class ToggleWidthComponent {
+  protected isDev: boolean;
+
+  constructor(platformUtilsService: PlatformUtilsService) {
+    this.isDev = platformUtilsService.isDev();
+  }
```
Comment (willmartian) [inline apps/web/src/app/layouts/toggle-width.component.ts:24]:
```suggestion
  protected isDev: boolean = inject(PlatformUtilsService).isDev()
```

Thoughts on favoring this syntax more generally in components?

**Review by willmartian — APPROVED** [review]
Nice!

File: apps/web/src/app/layouts/toggle-width.component.ts (hunk 1 above)
Comment (Hinton) [reply apps/web/src/app/layouts/toggle-width.component.ts:24]: I'm not against it, it does make things more angular specific though, not sure how all testing tools handles this?

**Review by coltonhurst — APPROVED** [review]
Thanks 👍

**Pull request merged**